src/components/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ReactComponent as Sign } from './sign.svg';
+import {
+  FooterStyled, LogoLink, FooterLink, FooterA,
+} from './Footer.style';
+
+function Footer() {
+  return (
+    <FooterStyled>
+      <FooterA href="https://ooloo.io" target="_blank">ooloo.io</FooterA>
```

Comment:
You can use FooterLink here as well together with the [as](https://styled-components.com/docs/api#as-polymorphic-prop) prop